usr/lib/ep11_stdll/ep11_specific.c

```diff
@@ -16,6 +16,9 @@
 #include <regex.h>
 #include <dirent.h>
 #include <libgen.h>
+#ifndef NO_PKEY
+#include <asm/pkey.h>
```

Comment:
For what exactly do you need this include here? Is it for `MAXPROTKEYSIZE` only ? If so, then you might not need that anymore when changing the length check as suggested below.